warehouse/query-core/src/main/java/datawave/query/jexl/functions/CompareFunctionValidator.java

```diff
@@ -31,14 +32,14 @@ public static void validate(String function, List<JexlNode> args) throws Illegal
                             MessageFormat.format("{0} requires 4 arguments", function));
             throw new IllegalArgumentException(qe);
         } else {
-            if (!operators.contains(args.get(1).image)) {
+            if (!operators.contains(String.valueOf(JexlNodes.getImage(args.get(1))))) {
```

Comment:
Can the image be null in which case String.valueOf(null) will return "null"?  That might not be what we want here.

Reply:
I made a comment in JexlNodes to add a getStringImage method that can handle this case appropriately.  Then we can drop all of the String.valueOf calls everywhere.

Reply:
The image CAN be null per my earlier comments.  I updated this to call JexlNodes.getIdentifierOrLiteralAsString which should match the previous behavior.